### PR TITLE
Add setEventUploadThreshold API

### LINKED ISSUE #40: Expose EventUploadThreshold property to Flutter
It is currently not possible to change EventUploadThreshold with the Flutter API.

### PR DESCRIPTION
**Pull request by spkersten**
Fixes https://github.com/amplitude/Amplitude-Flutter/issues/40

### CODE REVIEW

File: ios/Classes/SwiftAmplitudeFlutterPlugin.swift [hunk 1]
```diff
@@ -57,6 +57,10 @@ import Amplitude
                     let libraryVersion = args["libraryVersion"] as! String
                     Amplitude.instance(withName: instanceName).libraryVersion = libraryVersion
                     result(true)
+                case "setEventUploadThreshold":
+                    let eventUploadThreshold = args["eventUploadThreshold"] as! Int32
```
Comment (haoliu-amp) [inline ios/Classes/SwiftAmplitudeFlutterPlugin.swift:61]: I think we should use `Int` here

**Review by haoliu-amp — CHANGES_REQUESTED** [review]

**Comment by haoliu-amp** [comment]
@spkersten Thanks for your PR! Just some nits.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## 2.3.1
```
Comment (dantetam) [inline CHANGELOG.md:1]: ~~@haoliu-amp I remember I had to separate the version number from the commit, like in Amplitude/unity-plugin and also like this [previous commit](https://github.com/amplitude/Amplitude-Flutter/commit/b0b264dddaba7a72300fa57c26cf7c056c9bb5c9). Are we enforcing this here?~~ It's fine

File: ios/Classes/SwiftAmplitudeFlutterPlugin.swift (hunk 1 above)
Comment (spkersten) [reply ios/Classes/SwiftAmplitudeFlutterPlugin.swift:61]: And then make the conversion explicit on the next line with:
```
Amplitude.instance(withName: instanceName).eventUploadThreshold = Int32(eventUploadThreshold)
```
? What do you prefer?

Comment (haoliu-amp) [reply ios/Classes/SwiftAmplitudeFlutterPlugin.swift:61]: Oh you're right. I thought `eventUploadThreshold` is a NSInteger. Nvm then, this looks good!

**Review by haoliu-amp — APPROVED** [review]

**Comment by haoliu-amp** [comment]
@spkersten Thanks! https://pub.dev/packages/amplitude_flutter New version was released.

**Pull request merged**